app/nexus.go

```diff
@@ -267,6 +304,28 @@ func NewNexusCommand(getNexusClientFn GetNexusClientFn) (CommandOut, error) {
 		Usage:    "Namespace that is allowed to call this endpoint",
 		Required: true,
 	}
+	getEndpointDescription := func(ctx *cli.Context) (string, error) {
+		description := ctx.String(endpointDescriptionOptionalFlag.Name)
+		descriptionFile := ctx.String(endpointDescriptionFileOptionalFlag.Name)
+		if description != "" && descriptionFile != "" {
```

Comment:
we should also guard against somebody calling unset and also setting the description

Reply:
ignore